sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -439,6 +440,16 @@ class Analyzer(
             case s: Star => s.expand(child, resolver)
             case o => o :: Nil
           })
+        case p: Murmur3Hash if containsStar(p.children) =>
```

Comment:
I see. That is based on `lookupFunction` to determine if the inputs generated by star expansion can be accepted by the function.

When going over the function list we supported, very few of them need a support for star expansion. Another two functions are `greatest` and `least`. : )